modules/overview-of-accelerators.adoc

```diff
@@ -15,12 +15,14 @@ If you work with large data sets, you can use accelerators to optimize the perfo
 
 * NVIDIA graphics processing units (GPUs)
 ** To use compute-heavy workloads in your models, you can enable NVIDIA graphics processing units (GPUs) in {productname-short}. 
-**   To enable GPUs on OpenShift, you must install the NVIDIA GPU Operator.
+**   To enable GPUs on OpenShift, you must install the link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html[NVIDIA GPU Operator].
 * Habana Gaudi devices (HPUs)
 ** Habana, an Intel company, provides hardware accelerators intended for deep learning workloads. You can use Habana Gaudi accelerators on {productname-short} with version 1.10.0 of the Habana Gaudi Operator. 
 ** You can enable Habana Gaudi devices on-premises or with AWS DL1 compute nodes on an AWS instance.
 
-Before you can use an accelerator in {productname-short}, your OpenShift instance must contain an associated accelerator profile . For accelerators that are new to your deployment, you must manually configure an accelerator profile for the accelerator in context. If you use NVIDIA GPUs, an accelerator profile is automatically created after you upgrade to the latest version of {productname-short}.
+Before you can use an accelerator in {productname-short}, your OpenShift instance must contain an associated accelerator profile. For accelerators that are new to your deployment, you must manually configure an accelerator profile for the accelerator in context. If you use accelerators, an accelerator profile is automatically created after you upgrade to the latest version of {productname-short}.
+
+In addition, before you can successfully enable Habana Gaudi devices in {productname-short}, you must install the necessary dependencies and install the HabanaAI Operator. This allows your data scientists to use Habana libraries and software associated with Habana Gaudi devices from their notebooks. For more information about how to enable your OpenShift environment for Habana Gaudi devices, see link:https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/[HabanaAI Operator for OpenShift].
```

Comment:
Style points:
- Could remove 'In addition'
- We probably don't need the second 'install' here.
- Could avoid 'allows' by rewording to something like:

_This makes the Habana libraries and software associated with Habana Gaudi devices available from your notebook._

Reply:
Changed to your suggestions.